src/main/scala/uk/gov/nationalarchives/tdr/api/graphql/GraphQlTypes.scala

```diff
@@ -20,7 +20,8 @@ object GraphQlTypes {
       AntivirusMetadataFields.mutationFields ++
       FileMetadataFields.mutationFields ++
       FFIDMetadataFields.mutationFields ++
-      FinalTransferConfirmationFields.mutationFields
+      FinalTransferConfirmationFields.mutationFields ++
+      fields.FileStatusFields.mutationFields
```

Comment:
I think you can remove the `fields` prefix here as the whole object has been imported.

Reply:
Right, I have removed the `fields` prefix, thanks